### PR TITLE
Fix accessibility

### PR DESCRIPTION
**Pull request by stevenMouret**
- Fix aria-describedby when using data-parsley-error-message
- Manage aria-hidden attribute on error wrapper

### CODE REVIEW

File: src/parsley/ui.js [hunk 1]
```diff
@@ -170,16 +170,28 @@ UI.Field = {
...
+      var removeErrorMessage = this._ui.$errorsWrapper
         .removeClass('filled')
+        .attr('aria-hidden', 'true')
         .find('.parsley-custom-error-message')
         .remove();
+
+      var removeDescribedby = this._ui.$errorClassHandler
+        .removeAttr('aria-describedby');
+
+      return removeErrorMessage && removeDescribedby;
```
Comment (marcandre) [inline src/parsley/ui.js]: This return statement makes no sense to me. Simplest change would be to first `removeAttr('aria-describedby')` then do as before, no?

**Comment by marcandre** [comment]
Great, thanks for the PR. I commented on a `return`. Also, it would be great if this was part of one of the tests. Are you up to tweak this?

File: src/parsley/ui.js (hunk 1 above)
Comment (stevenMouret) [reply src/parsley/ui.js]: Bonjour Marc andré, 

I am not sure to understand. In the case there is no input assistance, it is necessary to remove aria-describedby. If we do not remove aria-describedby the technical assistance will try to read an empty ul.

Comment (marcandre) [reply src/parsley/ui.js]: Please re-read carefully what I wrote. Nowhere do I propose not removing the `aria-describedby`.

Comment (stevenMouret) [reply src/parsley/ui.js]: Hello Marc André,

Sorry this pull request is open for some time now.
Maybe it is possible to do more simple, but I am not a JS expert. The chunk of code you are highlighting just allow to remove the aria-describedby and the error message. If it is possible to make more simple do not hesitate to refactoring my code. If you need more explanation about the this accessibility changes I can explain more.

**Comment by marcandre** [comment]
I've pushed a modified commit to master, but I'm running into issues on Travis.

I've wasted an hour of my life on incompatibilities that seem to me [completely ridiculous](https://stackoverflow.com/questions/46053414/npm-warn-requires-a-peer-of-but-none-is-installed-you-must-install-peer) but it's still not working.

**Comment by stevenMouret** [comment]
> 
> 
> I've pushed a modified commit to master, but I'm running into issues on Travis.
> 
> I've wasted an hour of my life on incompatibilities that seem to me [completely ridiculous](https://stackoverflow.com/questions/46053414/npm-warn-requires-a-peer-of-but-none-is-installed-you-must-install-peer) but it's still not working.

I think you're getting the wrong issue. My changes do not affect dependencies.

**Comment by marcandre** [comment]
> My changes do not affect dependencies.

Indeed, there is no doubt about that.

> I think you're getting the wrong issue. 

No. I'm just explaining where I'm at. I have never thought or implied that your commit, or my modified version of it, was responsible for the troubles on Travis.

**Comment by marcandre** [comment]
Merged in with modifications as a1a5d36

Should be fixed in 2.9.2

Thanks again for the PR :+1:

**Pull request closed without merging**